<?xml version="1.0" ?>
<launch>
  <!-- config values used by more than one package so can't be private -->    
  <rosparam command="load" file="$(find pan_tilt)/config/common_config.yaml" />
  <rosparam command="load" file="$(find head_control)/config/common_config.yaml" /> 
  
  <node pkg="pan_tilt" type="pan_tilt_node" name="pan_tilt_node" output="screen">
    <rosparam command="load" file="$(find pan_tilt)/config/config.yaml" />
  </node>
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node" output="screen" args="/dev/ttyUSB0"/>
  <node pkg="head_control" type="head_control_node" name="head_control_node" output="screen">
    <rosparam command="load" file="$(find head_control)/config/config.yaml" />
  </node>
</launch>
